x-pack/plugins/observability_solution/apm/server/routes/dependencies/get_metadata_for_dependency.ts

```diff
@@ -46,16 +48,18 @@ export async function getMetadataForDependency({
           ],
         },
       },
+      fields: ['*'],
```

Comment:
if we can avoid using a wildcard here, that'd be great, for performance reasons.

Reply:
here we only need `SPAN_TYPE` and `SPAN_SUBTYPE`